docs/src/css/custom.css

```
@@ -6,18 +6,123 @@
+    --ifm-navbar-background-color: #f2f5f6; /* Navbar background color */
+    --ifm-navbar-link-color: #000e33;      /* Navbar link color */
+    --ifm-navbar-link-hover-color: #3c6de7; /* Navbar link hover color */
+    --ifm-mobile-menu-link-color: #8d959f;
+    --ifm-footer-background-color: #111927; /* Footer background color */
+    --ifm-primary-button-color : #1e57e2;
+    --ifm-primary-button-hover-color : #306cce;
+    --ifm-secondary-button-color : #eaecf0;
+    --ifm-secondary-button-hover-color : #d4d4d9;
+    --ifm-text-color: #ffffff;
+    --ifm-text-color-black: #000000;
+    -docusaurus-progress-bar-color: var(--ifm-color-primary);
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)